inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/single_layer/multiclass_nms.hpp

```diff
@@ -7,20 +7,22 @@
 #include <string>
 #include <tuple>
 
-#include "ngraph_functions/builders.hpp"
-#include "shared_test_classes/base/layer_test_utils.hpp"
+#include "ngraph_functions/utils/ngraph_helpers.hpp"
+#include "common_test_utils/common_utils.hpp"
+#include "shared_test_classes/base/ov_subgraph.hpp"
 
-namespace LayerTestsDefinitions {
+namespace ov {
+namespace test {
+namespace subgraph {
 
-using InputShapeParams = std::tuple<size_t,   // Number of batches
-                                    size_t,   // Number of boxes
-                                    size_t>;  // Number of classes
+// Dynamic shape + Target static shapes + output if static shape
+using ShapeParams = std::tuple<std::vector<ngraph::PartialShape>, std::vector<std::vector<ngraph::Shape>>, bool>;
```

Comment:
use `InputShape` to define dynamic and target shapes per each input
please see: https://github.com/openvinotoolkit/openvino/blob/89a3f4560227ab034b3698681ed610f3871265a6/inference-engine/tests/functional/plugin/cpu/single_layer_tests/variadic_split.cpp


also I guess we no need to pass `bool` which is `m_outStaticShape`
because we can define that outputs will be dynamic if one of input dims is dynamic

Reply:
Updated to use the InputShape for inputs.

If we want the output shape static I think we need to keep 'm_outStaticShape'.